src/style.css

```
@@ -1 +1,63 @@
 /* Styles go here */
+
+body,
+ul {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif ;
+  font-style: normal;
+  font-size: 12px;
+  text-transform: uppercase;
```

```suggestion
  text-transform: uppercase;
```
move it to `.nav__link`